src/main/java/appeng/client/render/RenderBlocksWorkaround.java

```diff
@@ -41,15 +42,15 @@
 public class RenderBlocksWorkaround extends RenderBlocks
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a constructor to the class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1258)